handbook/HR/maternity-policy.md

```diff
@@ -0,0 +1,184 @@
+
+We may ask you to provide the following, unless it is the first appointment:
+
+1. a certificate from the doctor, midwife or health visitor stating that you are pregnant; and
+2. an appointment card.
+
+Sickness
+
+Periods of pregnancy-related sickness absence shall be paid in accordance with the statutory sick pay scheme in the same manner as any other sickness absence.  Sick pay under your contract of employment is normally paid for up to 8 weeks absence in any 52 week period.  Any payment of sick pay in excess of this as a result of pregnancy-related sickness shall be entirely at our discretion.
+
+Periods of pregnancy-related sickness absence from the start of your pregnancy until the end of your maternity leave will be recorded separately from other sickness records and will be disregarded in any future employment-related decisions.
+
+If you are absent for a pregnancy-related reason during the four weeks before your Expected Week of Childbirth, your maternity leave will usually start automatically (see Paragraph to Paragraph ).
+
+Health and safety
+
+We have a general duty to take care of the health and safety of all employees. We are also required to carry out a risk assessment to assess the workplace risks to women who are pregnant, have given birth within the last 6 months or are still breastfeeding.
+
+We will provide you with information as to any risks identified in the risk assessment, and any preventive and protective measures that have been or will be taken.  If we consider that, as a new or expectant mother, you would be exposed to health hazards in carrying out your normal work we will take such steps as are necessary (for as long as they are necessary) to avoid those risks.  This may involve:
+
+1. changing your working conditions or hours of work;
+2. offering you suitable alternative work on terms and conditions that are the same or not substantially less favourable; or
+3. suspending you from duties, which will be on full pay unless you have unreasonably refused suitable alternative work.
+
+Entitlement to maternity leave
+
+All employees are entitled to up to 52 weeks&#39; maternity leave which is divided into:
+
+1. Ordinary Maternity Leave of 26 weeks (&quot;OML&quot;).
+2. Additional Maternity Leave of a further 26 weeks immediately following OML (&quot;AML&quot;).
+
+Starting maternity leave
+
+The earliest date you can start maternity leave is 11 weeks before the Expected Week of Childbirth (unless your child is born prematurely before that date).
+
+You must notify us of your Intended Start Date in accordance with Paragraph 3.  We will then write to you within 28 days to inform you of the date we will expect you to return to work if you take your full entitlement to maternity leave (&quot;Expected Return Date&quot;).
```

Comment:
Does CharlieHR have a process for this?

Reply:
You can book time off as "parental", but that's about it. If we have someone go on maternity leave I'll make sure to review the policy and manage the process timeline.